src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImplTest.java

```diff
@@ -111,14 +117,17 @@ public void setUp() {
         when(submissionServiceMock.getByKey(Mockito.eq(CASE_8)))
                 .thenReturn(Optional.of(submissionNoCsoAccount));
 
-        sut = new SubmissionApiDelegateImpl(submissionServiceMock, navigationProperties, cachePropertiesMock, submissionMapperMock, feeServiceMock, efilingAccountServiceMock);
+        ServiceFee serviceFee = new ServiceFee();
+        serviceFee.setFeeAmt(BigDecimal.valueOf(2));
```

Comment:
does 2 have a meaning or is it just a value

Reply:
Just the value of two. When using bigdecimal we can't just assign it an int